profiler/profiler.py

```diff
@@ -151,15 +149,17 @@ def main():
         required=False,
         help="The flamegraph output produced by prof2",
     )
-    args = parser.parse_args()
+    args = parser.parse_args(sys.argv[1:])
 
-    print(f"Process to be run: {args.process}")
-    # Running the main process using time -v to get detailed memory and time"""
+    print(f"Command to be run: {args.command}", file=stderr)
+    # Run the command, using `time -v` to get detailed memory and time"""
     p = subprocess.Popen(
-        ["/usr/bin/time", "-al"] + args.process, stdout=PIPE, stderr=PIPE
+        [args.gtime_cmd, "--format", GNU_TIME_FORMAT] + args.command.split(" "),
```

Comment:
Where is gtime command set and wondering why gnu time is a `/usr/bin/time` choice than `/usr/bin/time`

Reply:
Now that it's a cmd-line arg, it's up to the user to specify the path to gnu-time if it's not the default value (/usr/bin/time). The Click option for this param defines the default value.